application/controllers/collection_instrument.py

```diff
@@ -277,17 +278,43 @@ def unlink_instrument_from_exercise(self, instrument_id, exercise_id, session=No
             bound_logger.info("Failed to unlink, unable to find instrument or exercise")
             raise RasError("Unable to find instrument or exercise", 404)
 
-        instrument.exercises.remove(exercise)
-        for business in instrument.businesses:
```

Comment:
Removed as per the comment, unlinking is not something a SEFT should be doing (confirmed with Gareth) and as instrument businesses is a SEFT association it becomes redundant